src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/BrowserHttpHandler.cs

```diff
@@ -127,176 +125,190 @@ protected internal override HttpResponseMessage Send(HttpRequestMessage request,
             throw new PlatformNotSupportedException();
```

Comment:
Unrelated to this PR but I noticed that `Properties` uses a `ConcurrentDictionary` on `FEATURE_WASM_THREADS`. Why bother when even `SocketsHttpHandler` just uses a `Dictionary`?

https://github.com/dotnet/runtime/blob/669771100ca599e4d2355f1b3cf8bb34c6f0bdc6/src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/SocketsHttpHandler.cs#L409